src/components/layout/UserSwitchWrapper.tsx

```diff
@@ -30,7 +33,7 @@ const UserSwitchWrapper: FC<Props> = ({ ButtonContent }) => {
       <GraaspUserSwitch
         ButtonContent={ButtonContent}
         signOut={signOut}
-        currentMember={member}
+        currentMember={member as RecordOf<Member>}
```

Comment:
The correct type would be: (but it will change with the immutable removal, so we might wait for it merge and adapt after.
```suggestion
        currentMember={member as ImmutableCast<CompleteMember>}
```